toolkit/javascripts/clear-filters.js

```diff
@@ -0,0 +1,36 @@
+/* 
+  ClearFilters
+
+    Requirements
+      ID: #clearAllFilters 
```

Comment:
Looks like our precedent is to use hyphenation rather than camel case for IDs, so let's change this to `dm-clear-all-filters`.